api-mobile/src/swagger/swagger.yaml

```diff
@@ -1,911 +0,0 @@
-swagger: "2.0" # npm library `swagger-tools` does not yet support 3.0
```

Comment:
Definitely dont want to delete this file

Reply:
Oo. yeah. Don't know why that happened.

Reply:
I'll fix that first.

Reply:
Swagger is including in the top level .gitignore. That's weird.

Reply:
k... removed swagger from the `.gitignore`. It gets confusing having all these different copies.

Reply:
Oh thats weird